### DISCUSSION TITLE
I got on stealer by downloading files on the site

### DISCUSSION THREAD

**Discussion by Tolianbus** [community/community#120210] [Code Security] [answered]
### Select Topic Area

Question

### Body

https://github.com/olsenngt/Near-wallet-bot - file stealer
https://www.virustotal.com/gui/file/7bdbcdf440b9c25e34999ce2c5390e21b0fcf378524a942dde23c4ea92baebcd/detection - virus total file (bot.exe)

Please ban user, and delete file, this dungeous for all them

**Comment by queenofcorgis** [comment, marked as answer]
Hi @Tolianbus 👋🏼  We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our abuse reporting tools. Here's all the info:

- [Reporting abuse or spam](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam)
- [Reporting a user](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user)
- [Reporting an issue or pull request](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)
- [Reporting a comment](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-comment)

You can report behavior and content that violates community guidelines and terms. I am going to close this post, but for this and any future incidents, please refer to those links.

Thank you!
